sherpa-onnx/kotlin-api/OfflineRecognizer.kt

```diff
@@ -96,7 +99,10 @@ class OfflineRecognizer(
         val text = objArray[0] as String
         val tokens = objArray[1] as Array<String>
         val timestamps = objArray[2] as FloatArray
-        return OfflineRecognizerResult(text = text, tokens = tokens, timestamps = timestamps)
+        val lang = objArray[3] as String
+        val emotion = objArray[4] as String
+        val event = objArray[5] as String
+        return OfflineRecognizerResult(text = text, tokens = tokens, timestamps = timestamps, lang = lang, emotion = emotion, event = event)
```

Comment:
Could you please also update the java API at
https://github.com/k2-fsa/sherpa-onnx/blob/f300ec0f985904e577c0155e3aa87a7e57d58f70/sherpa-onnx/java-api/src/com/k2fsa/sherpa/onnx/OfflineRecognizerResult.java#L6-L9
?